pkgs/development/tools/rust/cargo-web/default.nix

```
@@ -13,6 +13,10 @@ rustPlatform.buildRustPackage rec {
     sha256 = "1dl5brj5fnmxmwl130v36lvy4j64igdpdvjwmxw3jgg2c6r6b7cd";
   };
 
+  patchPhase = ''
+    substituteInPlace src/utils.rs --replace-fail fs::File::open File::open
+  '';
+
```

```suggestion
  # https://github.com/koute/cargo-web/pull/275
  patches = [
    (fetchpatch {
      name = "fix-qualified-path.patch";
      url = "https://github.com/koute/cargo-web/commit/c9584542163d60d0aae6d6890509794e838e257f.patch";
      hash = "sha256-w59fXmrszptKt0llqGt0AF+0b3r9N6xUY6zQkpZnemE=";
    })
  ];
```

Could you use `patches` instead of `substituteInPlace'? The other contributor may feel the burden of checking if the patch is needed at that time. The clear reference makes it easy to update this package.